crmsh/bootstrap.py

```diff
@@ -1775,9 +1727,8 @@ def join_csync2(seed_host):
     # invoke scp root@seed_host:/etc/hosts $tmp_conf \
     #   || error "Can't retrieve /etc/hosts from seed_host"
     # install_tmp $tmp_conf /etc/hosts
-
-    if not invoke("scp root@%s:'/etc/csync2/{csync2.cfg,key_hagroup}' /etc/csync2" % (seed_host)):
-        error("Can't retrieve csync2 config from %s" % (seed_host))
+    if not invoke("%s root@%s:'/etc/csync2/{csync2.cfg,key_hagroup}' /etc/csync2" % (SCP_WITH_KEY, seed_host)):
```

Comment:
I did tired, but string format will take this `{csync2.cfg,key_hagroup}` as a value to replace and raise `KeyError: 'csync2'`.
use `\{csync2.cfg,key_hagroup\}` have the same error

Reply:
For that you need to put double `{{`. It would be: 
```
"{} root@{}:'/etc/csync2/{{csync2.cfg,key_hagroup}}' /etc/csync2" .format(SCP_WITH_KEY, seed_host)
```

Reply:
Thanks! Already changed!